lib/src/flow/bluetooth_provisioning_flow.dart

```diff
@@ -105,6 +113,30 @@ class _BluetoothProvisioningFlowState extends State<BluetoothProvisioningFlow> {
     );
   }
 
+  Future<void> _agentMinimumVersionDialog(BuildContext context, VoidCallback exitFunction) async {
+    await showDialog(
+      context: context,
+      barrierDismissible: false,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          title: const Text('Machine Incompatible'),
+          content: const Text(
+            'This machine\'s version is too low to connect via Bluetooth.\n\nPlease try a different provisioning method such as hotspot.',
```

Comment:
may need your input on the wording here, but the idea is they'll see this alert after connecting bluetooth ~> and then in CR we'll exit this bluetooth flow and go to hotspot @johnwalicki